servers/src/main/java/tachyon/worker/block/evictor/EvictorUtils.java

```diff
@@ -16,72 +16,92 @@
 package tachyon.worker.block.evictor;
 
 import java.io.IOException;
-import java.util.ArrayList;
-import java.util.Collections;
 import java.util.HashMap;
-import java.util.List;
 import java.util.Map;
 
 import tachyon.Pair;
 import tachyon.worker.block.BlockMetadataManager;
 import tachyon.worker.block.BlockStoreLocation;
 import tachyon.worker.block.meta.BlockMeta;
+import tachyon.worker.block.meta.StorageDir;
 
 public class EvictorUtils {
   /**
-   * Checks whether the plan is legal for an evictor with cascading eviction feature like
+   * Checks whether the plan is valid for an evictor with cascading eviction feature like
```

Comment:
I think this javadoc is not clear enough about what this method is doing. At least I didn't get it at the first time. How about changing it to: 

Checks whether a given plan with cascading eviction is valid. [and please explain what is cascading---I still don't have a good sense of this definition]. A plan may become invalid on cascading eviction if the total size of blocks to move to a lower tier is larger than the total amount of available space of this lower tier even after its blocks evicted.